### PR TITLE
feat: add future flag to True

### PR DESCRIPTION
**Pull request by dovdjian**
Add future flag to True to help cayzn to upgrade sqlalchemy to 2.0

### CODE REVIEW

File: remoulade/state/backends/postgres.py [hunk 1]
```diff
@@ -118,7 +118,7 @@ def __init__(
     ):
         self.url = url or os.getenv("REMOULADE_POSTGRESQL_URL") or DEFAULT_POSTGRES_URI
         super().__init__(namespace=namespace, encoder=encoder)
-        self.client = client or sessionmaker(create_engine(self.url, pool_pre_ping=True))
+        self.client = client or sessionmaker(create_engine(self.url, pool_pre_ping=True, future=True))
```
Comment (antoinerabany) [inline remoulade/state/backends/postgres.py:121]: You should pass it as argument  in __init__ I think

Comment (dovdjian) [reply remoulade/state/backends/postgres.py:121]: why I init ? In the doc they told about to add future flag in create_engine or sessionmaker

Comment (antoinerabany) [reply remoulade/state/backends/postgres.py:121]: in the init of PostgresBackend

Comment (dovdjian) [reply remoulade/state/backends/postgres.py:121]: ok I will do it

File: .bumpversion.cfg [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 1.2.3
+current_version = 1.2.4
```
Comment (antoinerabany) [inline .bumpversion.cfg:2]: no don't modify this, ill do it with bumpversion

File: remoulade/state/backends/postgres.py [hunk 3]
```diff
@@ -115,10 +115,11 @@ def __init__(
         client: Optional[sessionmaker] = None,
         max_size: int = 2000000,
         url: Optional[str] = None,
+        future: bool = True,
```
Comment (antoinerabany) [inline remoulade/state/backends/postgres.py:118]: default false to be non breaking ?

**Review by antoinerabany — CHANGES_REQUESTED** [review]

File: remoulade/__init__.py [hunk 4]
```diff
@@ -90,4 +90,4 @@
     "set_scheduler",
 ]
 
-__version__ = "1.2.3"
+__version__ = "1.2.4"
```
Comment (antoinerabany) [inline remoulade/__init__.py:93]: don't update this

**Comment by dovdjian** [comment]
c'est mieux maintenant ?

**Pull request merged**